sdk/src/main/java/com/meniga/sdk/models/sync/operators/MenigaSyncOperationsImp.java

```diff
@@ -8,12 +8,14 @@
 import com.meniga.sdk.webservices.requests.StartRealmSync;
 import com.meniga.sdk.webservices.requests.StartSync;
 
+import javax.annotation.Nullable;
+
 /**
  * Copyright 2017 Meniga Iceland Inc.
  */
 public final class MenigaSyncOperationsImp implements MenigaSyncOperations {
 	@Override
-	public Result<MenigaSync> startSync(Long realmUserId, long timeout) {
+	public Result<MenigaSync> startSync(Long realmUserId, @Nullable String sessionToken, long timeout) {
```

Comment:
I guess `realmUserId` can also be annotated (there's a null check in the method body):

```
public Result<MenigaSync> startSync(@Nullable Long realmUserId, @Nullable String sessionToken, long timeout) {
```

Reply:
Added